### PR TITLE
Add new Govspeak Preview component to 1.3 features (HTML attachments, edit edition withdrawal messages)

### PR DESCRIPTION
**Pull request by DilwoarH**
## What

We need to add the [previewable text area component](https://trello.com/c/FAig6Fn0/792-implement-previewable-text-area-for-govspeak "‌") on HTML attachments, edit edition and Withdrawal Messages.

## Why

So users can quickly preview Govspeak

https://trello.com/c/fB7EzPg5/817-add-new-govspeak-preview-component-to-13-features-html-attachments-edit-edition-withdrawal-messages

## Before

Screenshot 2022-10-27 at 3 18 13 pm

Screenshot 2022-10-27 at 3 28 29 pm

## After

Screenshot 2022-10-27 at 3 18 23 pm

screenshot-whitehall-admin dev gov uk-2022 10 27-15_18_48

Screenshot 2022-10-27 at 3 27 40 pm

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by davidgisbey** [comment]
lgtm but i think it needs to be added to withdrawal messages

**Comment by DilwoarH** [comment]
> lgtm but i think it needs to be added to withdrawal messages

ah yes thanks

File: app/views/admin/attachments/_form.html.erb [hunk 1]
```diff
@@ -41,7 +41,7 @@
         } %>
       </div>
 
-      <%= render "govuk_publishing_components/components/textarea", {
+      <%= render "components/govspeak-editor", {
```
Comment (davidgisbey) [inline app/views/admin/attachments/_form.html.erb:44]: Sorry to be a pain but i'm getting this on this page when i click the preview button 

image

Comment (DilwoarH) [reply app/views/admin/attachments/_form.html.erb:44]: Is this the edit page? or some other page?

Comment (DilwoarH) [reply app/views/admin/attachments/_form.html.erb:44]: Found the issue - pushing a fix

**Review by davidgisbey — APPROVED** [review]
Just tested it again and it's working nicely 👍

**Pull request merged**